src/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <h1>Leaderboard</h1>
+    <main>
+        <div class="left-main">
+            <div class="recent">
+                <h3>Recent scores</h3>
+                <button>Refresh</button>
+            </div>
+            <ul id="scores">
+                <li>Nabil: 100</li>
+                <li>Nabil: 100</li>
+                <li>Nabil: 100</li>
+                <li>Nabil: 100</li>
+                <li>Nabil: 100</li>
+            </ul>
+        </div>
```

Comment:
- [ ] Let's ensure to use semantic HTML. For the Recent Scores section, let's wrap it inside a `<section>` tag instead of a meaningless `<div>` tag 😎